ci-operator/step-registry/openshift/e2e/aws/cert-rotation/openshift-e2e-aws-cert-rotation-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: openshift-e2e-aws-cert-rotation
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: ipi-conf-manual-creds
+    - ref: aws-provision-cco-manual-users-static
+    - ref: ipi-conf-manual-creds-remove-unnecessary-creds
+    - chain: ipi-install
+    - chain: aws-provision-bastionhost
+    test:
+    - ref: openshift-e2e-aws-cert-rotation
+    post:
+    - ref: aws-deprovision-stacks
+    - ref: aws-deprovision-s3buckets
+    - ref: ipi-deprovision-deprovision
```

Comment:
I think also need to append `aws-deprovision-users-and-policies` step to deprovision the resources created in `aws-provision-cco-manual-users-static`